src/components/navigation/TheNavLarge.vue

```diff
@@ -5,7 +5,7 @@
       <v-spacer />
     </v-toolbar>
     <v-toolbar dense flat class="navbar-toolbar-center justify-center custom-not-clickable-large custom-background-transparent">
-      <the-canvas-tools class="custom-clickable"/>
+      <the-canvas-tools v-if="this.loadCanvas" class="custom-clickable"/>
```

Comment:
This variable can be used to prevent unauthorized users from viewing the canvas maybe?